File: DUMPFutsalTournament/Web/src/app/infrastructure/classes/match.ts
import { Team } from './team';
import { MatchEvent } from './matchevent'
import { MatchType } from '../enums/matchtype'

export class Match {
	matchId: number;
	timeOfMatch: Date;
	homeTeam: Team;
	awayTeam: Team;
	homeGoals: number;
	awayGoals: number;
	matchType: MatchType;
	matchEvents: MatchEvent[];
	isActive: boolean;

	minute: number;
	second: number;
}

File: DUMPFutsalTournament/Data/Entities/Match.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Security.Policy;
using System.Threading.Tasks;
using DUMPFutsalTournament.Data.Enums;

namespace DUMPFutsalTournament.Data.Entities
{
    public class Match
    {
        public int MatchId { get; set; }
        public DateTime TimeOfMatch { get; set; }
        public Team HomeTeam { get; set; }
        public Team AwayTeam { get; set; }
        public int HomeGoals { get; set; }
        public int AwayGoals { get; set; }
        public MatchType MatchType { get; set; }
        public ICollection<MatchEvent> MatchEvents { get; set; }
        public bool IsActive { get; set; }
    }
}
